cylc/flow/scripts/validate_reinstall.py

```diff
@@ -124,16 +125,16 @@ def check_tvars_and_workflow_stopped(
 
 @cli_function(get_option_parser)
 def main(parser: COP, options: 'Values', workflow_id: str):
-    sys.exit(vro_cli(parser, options, workflow_id))
+    sys.exit(asyncio.run(vro_cli(parser, options, workflow_id)))
 
 
-def vro_cli(parser: COP, options: 'Values', workflow_id: str):
+async def vro_cli(parser: COP, options: 'Values', workflow_id: str):
```

Comment:
Tangential (and my fault), but while we're here can we change this for consistency - VRO is dead as a name.
```suggestion
async def vr_cli(parser: COP, options: 'Values', workflow_id: str):
```